addons/binding/org.openhab.binding.kodi/ESH-INF/thing/thing-types.xml

```diff
@@ -110,6 +111,215 @@
 		<description>Sends a text to the Kodi Player to simulate keyboard text entry
 		</description>
 	</channel-type>
+	<channel-type id="inputexecuteaction" advanced="true">
+        <item-type>String</item-type>
+        <label>Execute an action</label>
+        <description>Execute a defined Input.Action within Kodi
```

Comment:
Omitt using `Input.Action` in the description. Please use a non (detailed) technical one.

Reply:
will do